cmd/provider-list/converter.go

```diff
@@ -72,3 +82,72 @@ func getProviderAndServiceName(name string) []string {
 	}
 	return nil
 }
+
+type ConverterParameters struct {
+	regorg        string
+	familyVersion string
+	monolith      string
+	refPackage    string
+}
+
+func (cc *ConverterParameters) ConfigurationMetadataV1(c *xpmetav1.Configuration) error {
+	var convertedList []xpmetav1.Dependency
+
+	for _, provider := range c.Spec.DependsOn {
+		if *provider.Provider == fmt.Sprintf("xpkg.upbound.io/upbound/%s", cc.monolith) {
+			continue
+		}
+		convertedList = append(convertedList, provider)
+	}
+
+	for ssopName := range SSOPNames {
```

Comment:
nit: We promote "family provider" instead of ssop, so we nay consider renaming this variable as `providerName`.:
```suggestion
	for providerName := range ProviderNames {
```

Reply:
Done!